mesheryctl/internal/cli/root/system/context/context.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Layer5, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package context
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/layer5io/meshery/models"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var configuration models.MesheryCtlConfig
+
+var (
+	availableSubcommands []*cobra.Command
+)
+
+// ContextCmd represents the update command
+var ContextCmd = &cobra.Command{
+	Use:   "context",
+	Short: "Switch between different meshery contexts",
+	Long:  `Switch between different meshery server version hosted on different URLs.`,
+	Args:  cobra.NoArgs,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if ok := utils.IsValidSubcommand(availableSubcommands, args[0]); !ok {
```

Comment:
It would be good to return the current context name when no subcommand is given.

Reply:
This might be a bit tricky. Can't we just have view for this ?

Reply:
Yes, we can have a `view` command, but there shouldn't be any technical hurdle in running the same logic that would be invoked under `view` when the user simply invokes `context` without any sub-command. Usage can also be provided.